server/app/views/admin/TranslationFormView.java

```diff
@@ -84,6 +85,12 @@ protected final FormTag renderTranslationForm(
       Locale locale,
       String formAction,
       ImmutableList<DomContent> formFieldContent) {
+    String redirectUrl = "";
+    if (formAction.contains("program")) {
```

Comment:
Is there any other way to determine the between program and question that doesn't rely on the formAction?

Reply:
changed it to accept a boolean instead (/*isProgramEdit=*/)